src/main/java/io/kestra/plugin/git/Clone.java

```diff
@@ -34,16 +34,24 @@
 @Plugin(
     examples = {
         @Example(
-            title = "Clone a repository from http server",
+            title = "Clone a public GitHub repository",
             code = {
-                "url: https://github.com/kestra-io/plugin-template",
-                "branch: develop",
-                "username: <username>",
-                "password: <password>"
+                "url: https://github.com/dbt-labs/jaffle_shop",
+                "branch: main",
+                "username: dbt-labs"
             }
         ),
         @Example(
-            title = "Clone a repository from ssh server",
+            title = "Clone a private repository from an HTTP server such as a private GitHub repository using a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)",
+            code = {
+                "url: https://github.com/anna-geller/kestra-flows",
+                "branch: main",
+                "username: anna-geller",
+                "password: your_personal_access_token"
+            }
+        ),
+        @Example(
+            title = "Clone a repository from an SSH server",
```

Comment:
Precise why you use directory

Reply:
wait I wasn't using a directory 😅 which line?

Reply:
I could have commented on a better line. Line 56, you use "directory: kestra", it will be better if, in your title, you explain what's his purpose.

Reply:
added